crates/api/src/peer_meta_store.rs

```diff
@@ -1,27 +1,48 @@
 use crate::{builder, config, BoxFut, K2Result, Timestamp, Url};
+use bytes::Bytes;
 use futures::future::BoxFuture;
 use std::sync::Arc;
 
 /// A store for peer metadata.
 ///
 /// This is expected to be backed by a key-value store that keys by space, peer URL and key.
 pub trait PeerMetaStore: 'static + Send + Sync + std::fmt::Debug {
-    /// Store a key-value pair for a given space and peer.
```

Comment:
I think removing this here makes sense but maybe it might still be worth having written somewhere (maybe in the top-level comment of the trait) that the `PeerMetaStore` is per space?

Reply:
If it is per space, that is. But I thought so.

Reply:
It is per space. Our understanding of how that was going to work was still evolving when the documentation was written but now every module except the transport is created per space. 

It wouldn't be a bad thing to document here but it seems like something more for top-level documentation somewhere

Reply:
Okay, I'm gonna resolve the conversation then.